docker-compose.yml

```diff
@@ -8,27 +8,46 @@ services:
             - .:/app
         depends_on:
             - db
+            - mongo
         ports:
             - "6543:6543"
     db:
         image: postgres:9.6
         restart: always
         container_name: eventme_postgres
         environment:
-          POSTGRES_USER: admin
-          POSTGRES_PASSWORD: 1
-          POSTGRES_DB: eventme
+            POSTGRES_USER: admin
+            POSTGRES_PASSWORD: 1
+            POSTGRES_DB: eventme
         volumes:
-          - ~/.eventme-postgres:/var/lib/postgresql/data
+            - ~/.eventme-postgres:/var/lib/postgresql/data
         ports:
-          - 5555:5432
+            - 5555:5432
+    mongo:
+        image: mongo
+        restart: always
+        container_name: mongo
+        ports:
+            - 27017:27017
+        volumes:
+            - ~/.eventme-mongo:/data/db
+    mongoUser:
```

Comment:
If you need this service only for adding user to connect MongoDB, you need to try using special env variables described in documentation to this Docker image: https://github.com/docker-library/docs/blob/master/mongo/README.md#-via-docker-stack-deploy-or-docker-compose

But as far I remember, MongoDB can be used without any creds defined. You may try this as well.

Reply:
I was trying to use env variables, but it's didn't work. Also, I was reading about using MongoDB without credentials, but adminer require the login credentials.

Reply:
That's weird and needs to be investigated. I checked adminer sources and there is case when creds are empty.

Reply:
Okay, I will try again